Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx

```diff
@@ -123,6 +123,7 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
       auto found = cellBuffer.find(currentIR);
       if (found == cellBuffer.end()) {
         currentCellContainer = std::make_shared<std::vector<Cell>>();
+        std::cout << "Setup cell container with " << currentCellContainer->size() << " entries" << std::endl;
```

Comment:
please use LOG instead of the std::cout

Reply:
Ooops, sorry, these are leftover from debugging, will remove them again ...